src/time_integration/methods_SSP.jl

```diff
@@ -26,12 +26,14 @@ The third-order SSP Runge-Kutta method of Shu and Osher.
 struct SimpleSSPRK33{StageCallbacks} <: SimpleAlgorithmSSP
     a::SVector{3, Float64}
     b::SVector{3, Float64}
```

Comment:
```suggestion
    numerator_a::SVector{3, Float64}
    numerator_b::SVector{3, Float64}
```
Maybe to avoid confusion when somebody sees that these values do not match the expected ones?